libs/core/src/lib/layout-grid/directives/layout-grid-row.directive.ts

```diff
@@ -0,0 +1,9 @@
+import { Directive } from '@angular/core';
+
+@Directive({
+    selector: '[fdLayoutGridRow]',
```

Comment:
Could you add also `fd-layout-grid-row` selector, just to keep same convention as on other directives?

Reply:
Done